info.csv

```diff
@@ -22,3 +22,4 @@ Rishabh Arora,18SCSE1010692,18021011915,Web Development,1,2,HTML CSS JavaScript,
 Monika Sharma,17SCSE101760,1713101727,Android Development,2,1,Java C++,Android development AI Cloud,https://github.com/Monika11sharma/HexOpCalci,monikasharma1899@gmail.com,https://github.com/Monika11sharma,https://www.linkedin.com/in/monika-sharma-666b28159/,Gryffindore
 Sahil Singh,17SCSE105021,1713105020,Android Development,2nd,Cloud 1,Java C,Android Development Cloud AI ML,NA,sahil91098singh@gmail.com,https://github.com/SahilMadridista,https://www.linkedin.com/in/sahil-singh-90b14b184/,NA
 Manish Kumar Vats, NA, NA, Android Development, 3rd, NA, "C++,Java,Game Development,VR", "Android Development, AR,VR development,Competitve Programming", NA, mvats77947794@gmail.com, https://github.com/Manish-Kumar-Vats, https://www.linkedin.com/in/manish-vats-505389146, "No Guts No Glory , No Legend No Story"
+Aryan,18SCSE1010395,18021011626,Cybersecurity Android Development Web development AI/ML,1,2,Java Python C Cybersecurity IOT Hardware,Smart-Mirror Smart-Speaker,3ncry9t3d@gmail.com,www,github.com/SKYR4,www.linkedin.com/skyr4,In this world of Binary,I m a 0 and u can be my 1

```

Comment:
You are missing Area of Expertise or Interest areas.
Apart from that, you can be enrolled in a single track only! So, Either it's gonna be Cyber Sec or Android or Web!

Reply:
I m editing my interest to a single track but I m not missing area of expertise or interest areas

Reply:
I edited the whole data, u can check again